.devcontainer/docker-compose.yml

```diff
@@ -44,6 +44,8 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_DB: postgres
       POSTGRES_PASSWORD: postgres
+    ports:
```

Comment:
is this necessary? not necessary on my machine

Reply:
interesting, it was breaking change on my machine lol. had to comment it out for devcontainer to build